stylesheets/design-patterns/_breadcrumb.scss

```diff
@@ -0,0 +1,43 @@
+.breadcrumb {
+
+  ol {
+    @extend %contain-floats;
+    padding-top: 0.75em;
+    padding-bottom: 0.75em;
+  }
+
+  li {
+    @include core-16;
+    float: left;
+
+    background-image: file-url("separator.png");
+    @include device-pixel-ratio() {
+      background-image: file-url("separator-2x.png");
+      background-size: 6px 11px;
+    }
+
+    background-position: 100% 50%;
+    background-repeat: no-repeat;
+
+    list-style: none;
+
+    margin-right: 0.5em;
+    margin-bottom: 0.4em;
+    margin-left: 0;
+    padding-right: 1em;
+
+    &:last-child {
```

Comment:
I'm working on a breadcrumb component in static, while working on it I spotted that this doesn't look great in IE8 and below because they dont support `last-child`, they do however support `first-child`.

See https://github.com/alphagov/static/compare/breadcrumb-component and https://github.com/alphagov/static/commit/dc60ba5f805bc0d9d8d0721cfe1e933ed2184d79